Application Settings/DependentPickList.page

```diff
@@ -0,0 +1,13 @@
+<apex:page Controller="DependentListController">
```

Comment:
Following changes need to be done
1. On page only two picklists are present. It looks so bad. Please add Labels to those picklist->
Select Country : its picklist
Select City : its picklist
2. There should be – as first value that is selected default in select country's picklist.(On load it looks weird because default value selected for country is India and still the country is not displayed)